### DISCUSSION TITLE
Version 1.5

### DISCUSSION THREAD

**Discussion by Alygnt** [Alygnt/Clifty#96] [Announcements]
## Clifty is much more powerful now!!

# Features :
## ADDED :
- Back option in the menu
- NOTP, POTP, and OOTP in sites
- Email
- Homepage
- New tunnel option - Serveo

## OPTIMISED :
- Main colors
- Color codes
- Spacing
- Fetch link
- Update
- Fetching data 

## FIXED :
- Logs error fixed
- Sites fixed
- OTP copies
-  NGROK 
-  Cloudlflared
-  Localxpose

## REMOVED :
- Shorten link APIs which are no longer working
- email NOTP, POTP, and OOTP files to sites

# BUGS FIXED :
#93 #37 #94 #92 #90 #91 #21 #35 #49 #53 #64 #79 #82 #86 #81 #74 #38 #60 #88 #39 #77 #40 #83 

# SITES
## ADDED
-  Camera
-  Location
-  Microphone
-  Clipboard
-  Audio
-  IP
-  Device details

## OPTIMISED
- Few sites

### **[View Change log ](https://github.com/Alygnt/Clifty/compare/v_1.4...v_1.5)**

<hr /><em>This discussion was created from the release <a href='https://github.com/Alygnt/Clifty/releases/tag/v_1.5'>Version 1.5</a>.</em>

**Comment by Storelawiss540** [comment]
whare camera save pictor
